qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -177,25 +178,33 @@ def process_sample_template(self, study, user, callback):
             # The file does not exist, fail nicely
             raise HTTPError(404, "This file doesn't exist: %s" % fp_rsp)
 
+        # Define here the message and message level in case of success
```

Comment:
It looks like I end up removing two lines that I shouldn't. I've re-added the lines that define the message and message level in case of success.